### ISSUE TITLE
Adding url http://prometheus:9090 for Data Source   - Error

### ISSUE THREAD

**Issue by rizekmWWT** [grafana/tutorials#88]
Adding url http://prometheus:9090 for Data Source - when Save & Test is clicked an "HTTP Error Bad Gateway" is received

**Comment by marcusolsson** [comment]
Thanks for reporting this! I've had indications that the Prometheus versions needed to be updated. I've bumped the versions. Could you run `git pull` on the `tutorials-environment` repository and try again to see if it works for you?

**Comment by rizekmWWT** [comment]
Thanks Marcus.  This resolved the Prometheus data source issue for me.  One other issue that I had to correct so far is that the tutorial app and Grafana run on the same 3000 port.  So I changed the tutorial app in docker-compose.yml to use port 3001 and I no longer receive port conflicts during the tutorial build process.

Thanks again,
Mike

Mike Rizek
Implementation Engineer | AppDynamics
World Wide Technology, Inc.  Sr. Consultant
